prime-router/metadata/schemas/covid-19.schema

```diff
@@ -591,6 +589,18 @@ elements:
     hl7Field: AOE
     hl7AOEQuestion: 82810-3
 
+  - name: processing_mode_code
+    type: CODE
+    valueSet: hl70103
+    default: P
+    hl7Field: MSH-11-1
+    documentation: P, D, or T for Production, Debugging, or Training
+
+  - name: message_id
+    type: ID
+    hl7Field: MSH-10
+    documentation: unique id to track the usage of the message
+
   # name: public_health_case_id
```

Comment:
Will want SimpleReport to send us their internal message id

Reply:
Definitely, and I like the processing_mode_code too.     The schema that's for Pima needs the code for the mode. ;)